File: project-euler/problem_1.rb
$i = 0
$num = 1000
numbers = []

while $i < $num do
  if $i%3==0 || $i%5==0
    numbers.push($i)
  end
  $i+=1
end

$total = 0
$j = 0

while $j < numbers.length do
  $total = $total + numbers[$j]
  $j += 1
end

print $total

File: project-euler/problem_1.js
numbers = []
for (i=0; i<1000; i++) {
  if(i%3==0 || i%5==0) {
    numbers.push(i);
  }
}
total = 0;
for (i=0; i<numbers.length; i++) {
  total = total + numbers[i]
  }
  console.log(total)
